packages/kbn-interpreter/package.json

```diff
@@ -7,6 +7,16 @@
     "canvas:peg": "pegjs common/lib/grammar.peg",
     "kbn:bootstrap": "yarn build"
   },
+  "kibana": {
```

Comment:
__issue:__ looks like it's a leftover? `target` folder will be deleted by default, and `extraPaths` isn't known option (vs `extraPatterns`)....

Reply:
Yeah, this is leftover, I had hoped to get this in before migrating the build stuff for interpreter and use it to clean up the old build artifacts from the previous build too.